src/demo/pages/DemoContainer.jsx

```diff
@@ -344,6 +345,190 @@ class DemoContainer extends React.Component {
                 </CTA>
               )}
             />
+            <h3 id="layout-components-form-layout" className="typography-size-4 mb-6">Form Layout</h3>
+            <p>
+              Vertical Form Layout works similar to List except that List Items are not needed.
+              Vertical form field layout is forced.
+            </p>
+            <Documentation
+              name="Vertical Form Layout"
+              component={(
+                <FormLayout>
+                  <>
```

Comment:
This is an example of `React.Fragment` that we need to be able to traverse for the `Form Layout` to work even with form fields nested in fragments.